scripts/setup_ebpf_cicd_tests.ps1

```diff
@@ -30,8 +33,11 @@ Remove-Item ".\TestLogs" -Recurse -Confirm:$false -ErrorAction SilentlyContinue
 # Get all VMs to ready state.
 Initialize-AllVMs -VMList $VMList -ErrorAction Stop
 
-# Download the release artifacts for regression tests.
-Get-RegressionTestArtifacts
+if ($TestMode -eq "CI/CD") {
+
+    # Download the release artifacts for regression tests.
+    Get-RegressionTestArtifacts
+}
 
 Get-Duonic
```

Comment:
this is only needed for xp and conn_redir tests

Reply:
Ah. I remember now why I kept it this way.

First off, `Initialize-NetworkInterfacesOnVMs` (called a bit later) requires this. 

Secondly, this allows using `duonic` for future 'km stress' tests that may _also_ need this sort of support.  Keeping it this way saves us from having to modify this script yet again when we _do_ need this kind of support for those additional (sure to come) km stress tests.  IMO this is a small cost to pay for the benefit of never having to think of this again :-)

Reply:
Yes, the call to  `Initialize-NetworkInterfacesOnVMs` below is also only necessary for xdp and conn_redir tests.
i agree that since stress and perf tests only run at night, this isn't a huge problem, but I don't agree that adjusting the script for a hypothetical stress test that does require Duonic is a problem either.